src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -41,6 +42,17 @@ def load_arguments(self, _):
          'SasDefinitionAttributes', 'SecretAttributes', 'CertificateAttributes', 'StorageAccountAttributes',
          resource_type=ResourceType.DATA_KEYVAULT)
 
+    class CLIJsonWebKeyOperation(str, Enum):
+        encrypt = "encrypt"
+        decrypt = "decrypt"
+        sign = "sign"
+        verify = "verify"
+        wrap_key = "wrapKey"
+        unwrap_key = "unwrapKey"
+        import_ = "import"
```

Comment:
Is this a typo?

Reply:
@jiasli Do you mean `import_`? No, it's not a typo, since `import` is a Python keyword so we can't use it as a variable name.

Reply:
🚢

Reply:
BTW, this is mentioned in [PEP 8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles).